src/Templates/src/templates/maui-mobile/MauiApp1/MauiApp1.csproj

```diff
@@ -3,7 +3,11 @@
 	<PropertyGroup>
 		<!-- iOS, Android, MacCatalyst -->
 		<TargetFrameworks>net6.0-ios;net6.0-android;net6.0-maccatalyst</TargetFrameworks>
+		<!-- Windows -->
+		<TargetFrameworks Condition="$(OS.Contains('Windows'))">net6.0-windows10.0.19041;$(TargetFrameworks)</TargetFrameworks>
 		<OutputType>Exe</OutputType>
+		<OutputType Condition="$(TargetFramework.Contains('-windows'))">WinExe</OutputType>
```

Comment:
A couple of targets in WinUI assume Exe == C++ and WinExe is C#. Not sure if that is good or bad. We might be able to adjust some targets or put this in a props file.